### ISSUE TITLE
mobile phone warning should have a close option

### ISSUE THREAD

**Issue by Nick-Tallguy** [hotosm/tasking-manager#673]
I often review tasks on my way to a mapathon I will be teaching at - using my phone on the train.

The warning to mobile phone users is good but there should be an option to hide it, perhaps on a per user basis.

Typed on my phone on a train!

**Comment by bgirardot** [comment]
@LindaAlblas @felskia Can we add this easily?

It being based on screen size also causes an issue. I could not demo using an overhead projector because it forced my screen size to something that caused this to display.

**Comment by russdeffner** [comment]
I would say this is a priority, if say I need to publish a disaster project on the road even if it's tough like it was on mobile with TM2, let me struggle my way but I don't see any way to get that done on a phone now; at least on iOS I couldn't figure out how to log in or anything.

**Comment by verdy-p** [comment]
See also bug #820.

**Comment by harry-wood** [comment]
I started looking into this but actually [this line](https://github.com/hotosm/tasking-manager/blob/develop/client/index.html#L118) is supposed to be showing an "X" button, and the functionality for dismissing the banner is all in place already. I think a style bug is meaning that it's not showing up.

**Comment by pgiraud** [comment]
@harry-wood it looks like the CSS rule `hot-ds-icon-xmark` is never defined.
And unfortunately this kind of commit messages doesn't really help https://github.com/hotosm/tasking-manager/pull/413

**Comment by pgiraud** [comment]
It looks like the different css rules prefixed with `hot-ds-` (actually used in several places in the code) aren't declared anymore. They were previously retrieved from https://github.com/hotosm/hot-design-system but this is not the case anymore.
https://github.com/hotosm/tasking-manager/commit/164f58cbaf9d59f3c3e8c6c57fcb26e5791a6925#diff-05d76e6130fce41e9f34b6a1aa08bd50L18

@felskia Should we replace the hot-ds-xxx classes with something else?

**Comment by ethan-nelson** [comment]
For reference, here are the remaining hot-ds classes in case someone is interested in working on those: 

```
server/web/static/dist/app/components/account-nav/account-nav.html:    <i class="hot-ds-icon-sm-xmark pull-right pointer" ng-click="accountNavCtrl.hideNotification()"></i>
server/web/static/dist/app/project/project.html:                                           class="hot-ds-icon-circle-question pointer">
server/web/static/dist/app/project/project.html:                                           class="hot-ds-icon-circle-question pointer">
server/web/static/dist/app/admin/edit-project/edit-project.html:                                            {{ user }} <i class="hot-ds-icon-sm-xmark" ng-click="editProjectCtrl.removeUser(user)"></i>
server/web/static/dist/app/profile/profile.html:                                <i class="hot-ds-icon-pencil"></i> {{ 'Project Manager' | translate }}
server/web/static/dist/app/profile/profile.html:                                <i class="hot-ds-icon-pencil"></i> {{ 'Administrator' | translate }}
server/web/static/dist/app/profile/profile.html:                                            <i class="hot-ds-icon-sm-tick"></i>
server/web/static/dist/app/profile/profile.html:                                        <i class="hot-ds-icon-pencil" title="Edit Twitter URL"></i>
server/web/static/dist/app/profile/profile.html:                                        <i class="hot-ds-icon-tick" title="Save edits"></i>
```

**Comment by ethan-nelson** [comment]
This was fixed in #946 and is live.
